libkbfs/key_bundle_v2.go

```diff
@@ -20,15 +20,14 @@ import (
 // TLF's symmetric secret key information.
 type DeviceKeyInfoMapV2 map[keybase1.KID]TLFCryptKeyInfo
 
-func (dkimV2 DeviceKeyInfoMapV2) fillInDeviceInfo(crypto Crypto,
+func (dkimV2 DeviceKeyInfoMapV2) fillInDeviceInfos(crypto cryptoPure,
 	uid keybase1.UID, tlfCryptKey kbfscrypto.TLFCryptKey,
 	ePrivKey kbfscrypto.TLFEphemeralPrivateKey, ePubIndex int,
-	publicKeys []kbfscrypto.CryptPublicKey) (
-	serverMap map[keybase1.KID]kbfscrypto.TLFCryptKeyServerHalf,
-	err error) {
-	serverMap = make(map[keybase1.KID]kbfscrypto.TLFCryptKeyServerHalf)
+	publicKeys map[kbfscrypto.CryptPublicKey]bool) (
```

Comment:
Yes, I think so. In the old code, this was a list, but if there were duplicates, the first instance would fill in, and then all the others would be skipped because the first instance would put the right key in the map. Now, we just convert from list to set when we get the keys from the keybase daemon, so it just behaves the same.